packages/flutter/lib/src/material/carousel.dart

```diff
@@ -290,6 +292,14 @@ class CarouselView extends StatefulWidget {
   /// Called when one of the [children] is tapped.
   final ValueChanged<int>? onTap;
 
+  /// Determines whether an [InkWell] will cover each Carousel item.
+  ///
+  /// If true (the default), tapping an item will create an ink splash
+  /// as defined by the [ThemeData.splashFactory].
+  ///
+  /// Setting this to false allows the [children] to respond to user gestures.
```

Comment:
```suggestion
  /// Setting this to false allows the [children] to respond to user gestures.
  ///
  /// Defaults to true.
```

Reply:
Thank you very much for the suggestion!

Reply:
Oh, my bad—[the previous wording](https://github.com/flutter/flutter/pull/155214#discussion_r1777370648) was per my suggestion 😅

Both versions LGTM!

Reply:
Oh I didn't notice it. Sorry about that🤣. Since the documents for other properties mention the default value in the last paragraph haha, just for consistency. Agree that both are good:)!